metadata.yaml

```diff
@@ -12,6 +12,9 @@ resources:
     description: 'Backing OCI image'
     auto-fetch: true
     upstream-source: docker.io/kubeflownotebookswg/centraldashboard:v1.7.0
+provides:
+  sidebar:
```

Comment:
can we get this defined in `charm-relation-interfaces` and consider naming it something specific like `kubeflow_sidebar`?